packages/remediations/src/steps/reviewSystems.js

```diff
@@ -21,15 +21,23 @@ import { Provider, useDispatch, useSelector } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { ExclamationCircleIcon } from '@patternfly/react-icons';
 import './reviewSystems.scss';
+import isEqual from 'lodash/isEqual';
 
 const ReviewSystems = ({ issues, systems, registry, ...props }) => {
+
     let dispatch = useDispatch();
     const inventory = useRef(null);
     const { input } = useFieldApi(props);
     const formOptions = useFormApi();
-
+    const selectedRef = useRef();
     const inventoryApi = useRef({});
 
+    const { selected, loaded, rows } = useSelector(({ entities }) => ({
+        selected: entities?.selected || [],
+        loaded: entities?.loaded,
+        rows: entities?.rows || []
+    }));
```

Comment:
shallow equal

Reply:
This one will be tricky, `rows` are complicated value so that wouldn't really help. `selected` on other hand is array so `shallowEqual` would not help. But splitting this to specific selectors and using some comparators would not hurt.